### ISSUE TITLE
I sync bank - but dont see any transactions moving in the days after....

### ISSUE THREAD

**Issue by johndpope** [plaid/pattern#209]
1. <img width="1578" alt="Screen Shot 2022-02-08 at 2 10 01 pm" src="https://user-images.githubusercontent.com/289994/152910905-64de14d4-ee5d-4edf-91ba-e80b37915e05.png">

There's no manual override to sync? I assumed I'd see updates once a day if there were transactions that moved.
Is there something that would stop the link working?  

2. "Associating users with Plaid items and access tokens
Plaid does not have a user data object for tying multiple items together, so it is up to application developers to define that relationship. For an example of this, see the [root items route](https://github.com/plaid/pattern/blob/master/server/routes/items.js) (used to store new items) and the [users routes](https://github.com/plaid/pattern/blob/master/server/routes/users.js)."

I don't fully under the above from readme - if I have many users that sign into their own banks - would this have an impact on 1 ?

**Comment by lindawoo-plaid** [comment]
@johndpope, I do not quite understand what you are asking in #1.  What is your set-up?  I presume you are in the development environment and not using ngrok?  Normally, yes, if there were transactions that were added or removed, you'd receive a DEFALUT_UPDATE webhook or TRANSACTIONS_REMOVED webhook.  Plaid checks for these several times a day. 

With regard to #2, no, having many users and items for those users would not have an impact on transactions updates.  The readme is trying to say that in your own database, you need to link an item and access token to a particular user.  And if a user adds another item, you will also tie that item and access token to that user in your database.

**Comment by johndpope** [comment]
actually using  production secrets. I've signed in successfully with chase account - I move money by purchasing something - and I'm expecting plaid to broadcast these transactions....

I'm checking the logs on plaid.com - I get this - strangley the environment is saying sandbox.

https://plaid.com/docs/errors/item/#item_login_required

I've checked my code - and this has me a bit stumped. I'll raise an internal support ticket - case 393593
